lib/internal/bootstrap_node.js

```diff
@@ -268,15 +268,29 @@
     var console;
     Object.defineProperty(global, 'console', {
       configurable: true,
-      enumerable: true,
+      enumerable: false,
       get: function() {
         if (!console) {
           console = NativeModule.require('console');
           installInspectorConsoleIfNeeded(console,
                                           inspectorConsole,
                                           wrapConsoleCall);
         }
+        Object.defineProperty(global, 'console', {
+          configurable: true,
+          writable: true,
+          enumerable: false,
+          value: console
+        });
```

Comment:
Fail comes form 
```
=== release console_low_stack_space ===                    
Path: message/console_low_stack_space
Hello, World!
assert.js:86
  throw new assert.AssertionError({
```
I tried common.allowGlobals('console') in that file and 2 I create, it didn't help. Maybe I do something wrong.